package.yaml

```diff
@@ -15,7 +15,7 @@ homepage: http://haskellstack.org
 custom-setup:
   dependencies:
   - base
-  - Cabal
+  - Cabal == 2.*
```

Comment:
At least it didn't work on my machine with an older `Cabal`, but feel free to skip this change.